src/hupper/ipc.py

```diff
@@ -252,18 +244,7 @@ def get_command_line(**kwds):
     prog %= ', '.join('%s=%r' % item for item in kwds.items())
     opts = args_from_interpreter_flags()
     args = [sys.executable] + opts + ['-c', prog]
-
-    # ensure hupper is on the PYTHONPATH in the worker process
-    self_path = os.path.abspath(imp.find_module('hupper')[1])
-    extra_py_paths = [os.path.dirname(self_path)]
-
-    env = os.environ.copy()
-    env['PYTHONPATH'] = (
-        os.pathsep.join(extra_py_paths)
-        + os.pathsep
-        + env.get('PYTHONPATH', '')
-    )
-    return args, env
+    return args
```

Comment:
we set `sys.path` in the subprocess so this should be unnecessary

Reply:
oops found fd77382d31c287ce8d8b5a8969f63f9b4ec9fe60 explaining why it's necessary - gotta bring that back